test/cri-containerd/jobcontainer_test.go

```diff
@@ -0,0 +1,87 @@
+// +build functional
+
+package cri_containerd
+
+import (
+	"context"
+	"strings"
+	"testing"
+	"time"
+
+	runtime "k8s.io/cri-api/pkg/apis/runtime/v1alpha2"
+)
+
+func getJobContainerPodRequestWCOW(t *testing.T) *runtime.RunPodSandboxRequest {
+	return &runtime.RunPodSandboxRequest{
+		Config: &runtime.PodSandboxConfig{
+			Metadata: &runtime.PodSandboxMetadata{
+				Name:      t.Name(),
+				Namespace: testNamespace,
+			},
+			Annotations: map[string]string{
+				"microsoft.com/hostprocess-container": "true",
+			},
+		},
+		RuntimeHandler: wcowProcessRuntimeHandler,
+	}
+}
+
+func getJobContainerRequestWCOW(t *testing.T, podID string, podConfig *runtime.PodSandboxConfig) *runtime.CreateContainerRequest {
+	return &runtime.CreateContainerRequest{
+		Config: &runtime.ContainerConfig{
+			Metadata: &runtime.ContainerMetadata{
+				Name: t.Name() + "-Container",
+			},
+			Image: &runtime.ImageSpec{
+				Image: imageWindowsNanoserver,
+			},
+			Command: []string{
+				"cmd",
+				"/c",
+				"ping",
+				"-t",
+				"127.0.0.1",
+			},
+
+			Annotations: map[string]string{
+				"microsoft.com/hostprocess":              "true",
+				"microsoft.com/hostprocess-inherit-user": "true",
+			},
+		},
+		PodSandboxId:  podID,
+		SandboxConfig: podConfig,
+	}
+}
+
+func Test_RunContainer_Username_JobContainer_WCOW(t *testing.T) {
+	requireFeatures(t, featureWCOWProcess, featureHostProcess)
+
+	pullRequiredImages(t, []string{imageWindowsNanoserver})
+	client := newTestRuntimeClient(t)
+
+	username := "nt authority\\system"
```

Comment:
We don't actually pass any username through for the container to use, so we're just testing what the default identity is. Naming this test "*_Username_*" seems misleading.

Reply:
Changed to `*_InheritUser_*` to make this more explicit